### PR TITLE
#419: Accessibility report

### PR DESCRIPTION
**Pull request by RobLoach**
Based off of the updated build process from https://github.com/kalamuna/kalastatic/pull/97 .

#104

### CODE REVIEW

**Comment by andrewmallis** [comment]
@RobLoach would love to be able to merge this in for demo. Looks like maybe only small changes needed. Possible?

**Comment by RobLoach** [comment]
May have a look at this later on tonight.

File: package.json [hunk 1]
```diff
@@ -9,22 +9,24 @@
...
     "start": "grunt",
-    "styleguide": "./node_modules/kss-bootstrap/node_modules/kss/bin/kss-node --config='kss-node.json'",
-    "deploy": "grunt deploy"
+    "postinstall": "bower install && find node_modules/ -name '*.info' -type f -delete",
+    "deploy": "grunt deploy",
+    "kss": "./node_modules/kss-bootstrap/node_modules/kss/bin/kss-node --config='kss-node.json'",
+    "a11y": "a11y build/**/*.html > build/styleguide/a11y.txt"
   },
   "devDependencies": {
+    "a11y": "git://github.com/RobLoach/a11y.git#glob",
```
Comment (andrewmallis) [inline package.json]: fork to kalamuna? do we need a fork over the main project?

Comment (RobLoach) [inline package.json]: The required change was merged into the project: https://github.com/addyosmani/a11y/pull/37

Comment (andrewmallis) [inline package.json]: Great work, rob#glob.

**Comment by RobLoach** [comment]
I'd like to clean up the report so that it's pretty and nice looking in HTML.

**Comment by RobLoach** [comment]
https://github.com/kalamuna/kalastatic/pull/261

**Pull request closed without merging**